src/ripple/app/main/Main.cpp

```diff
@@ -704,10 +705,12 @@ run(int argc, char** argv)
     {
         try
         {
-            config->VALIDATION_QUORUM = vm["quorum"].as<std::size_t>();
-            if (config->VALIDATION_QUORUM == std::size_t{})
+            config->VALIDATION_QUORUM = vm["quorum"].as<std::int64_t>();
+            if (*config->VALIDATION_QUORUM < 1)
```

Comment:
Should this be `< 0`? The PR says reject negative and the original check is for negative.

Reply:
I don't think so: a quorum value of `0` doesn't make sense; notice, too, that the original code checked if the value was `0`. 
With that said, I would suggest replacing this code with `lexicalCast`:

```
try
{
    config->VALIDATION_QUORUM = lexicalCast<std::size_t>(vm["quorum"].as<std::string>());

    if (config->VALIDATION_QUORUM == 0)
        throw std::domain_error(s);
}
/* existing catch handlers */
```

Also, this bug is present because `boost::program_options::variable_value::as()` is broken. Per the [documentation](https://www.boost.org/doc/libs/1_56_0/doc/html/boost/program_options/variable_value.html#idp219097696-bb), it should be throwing a `boost::bad_any_cast` exception if a negative value is passed, but it doesn't. So I think this merits a bug report, if anyone feels s inclined.